spec/go_runner_spec.vim

```diff
@@ -0,0 +1,38 @@
+source spec/support/helpers.vim
+
+describe "Go Runner"
+  before
+    cd spec/fixtures
+  end
+  
+  after
+    cd -
+  end
+
+  describe "when test#go#runner is not set"
+    after
+      call Teardown()
+    end
+    it "should use ginkgo if 'github.com/onsi/ginkgo' is found"
+      view ginkgo/normal_test.go
```

Comment:
I wanted to have a unittest on `determine_runner` only, but I found that I have to vie w the file to make `search` work. Is there a better way?